Google.Api.Gax.Grpc/Rest/RestMethod.cs

```diff
@@ -142,11 +142,14 @@ internal HttpRequestMessage CreateRequest(IMessage protoRequest, string host)
             
             var uri = host is null ? new Uri(uriPathWithParams, UriKind.Relative) : new UriBuilder { Host = host, Path = uriPathWithParams }.Uri;
             
+            var content = transcodingResult.Body is null ? null : new StringContent(transcodingResult.Body, Encoding.UTF8, s_applicationJsonMediaType);
+            
             return new HttpRequestMessage
             {
+                Headers = { { HttpRequestHeader.ContentType.ToString(), "application/json" } },
```

Comment:
I don't *think* you need this - the content type header is logically part of the content, and I believe it will be set automatically with the way we're constructing the StringContent... unless we don't *have* any content, in which case I don't think we should be setting the content type anyway.

Reply:
👍